test/e2e/auth/pod_security_policy.go

```diff
@@ -186,6 +186,12 @@ func testPrivilegedPods(tester func(pod *v1.Pod)) {
 		sysadmin.Spec.Containers[0].SecurityContext.AllowPrivilegeEscalation = nil
 		tester(sysadmin)
 	})
+
+	By("Running RunAsGroup and RunAsUser")
+	privileged := restrictedPod("RunAsGroup")
+	privileged.Spec.Containers[0].SecurityContext.RunAsGroup = 65529
+	privileged.Spec.Containers[0].SecurityContext.RunAsUser = 0
+	tester(privileged)
```

Comment:
it would be two different cases one with RunAsGroup as 65529 and another with RunAsUser as 0

Reply:
two different By() blocks